analytics_data_api/utils.py

```diff
@@ -47,49 +45,6 @@ def matching_tuple(answer):
     )
 
 
-def consolidate_answers(problem):
```

Comment:
I moved this into `views/problems.py`.  It both seemed odd in utils, and I was running into some very strange test failures in non-related db tests when importing `utils.py` into `apps.py` when this method remained here.